src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -1242,6 +1297,49 @@ end = struct
           ~f:
             (Fn.compose add_endoscale_scalar_round
                (Endoscale_scalar_round.map ~f:reduce_to_v) )
+    | Plonk_constraint.T
+        (RangeCheck0
+          { v
+          ; vp0
+          ; vp1
+          ; vp2
+          ; vp3
+          ; vp4
+          ; vp5
+          ; vc0
+          ; vc1
+          ; vc2
+          ; vc3
+          ; vc4
+          ; vc5
+          ; vc6
+          ; vc7
+          ; compact
+          } ) ->
+        (*
+        //! 0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
+        //! v vp0 vp1 vp2 vp3 vp4 vp5 vc0 vc1 vc2 vc3 vc4 vc5 vc6 vc7
+        *)
+        let vars =
+          [| Some (reduce_to_v v)
```

Comment:
Why do you use parenthesis here? Is that necessary in OCaml?

Reply:
Yes, it means do that first, otherwise It might think it's `Some` with two parameters `recuce_to_v` and `v`